runtime/near-vm-logic/Cargo.toml

```diff
@@ -19,6 +19,8 @@ base64 = "0.11"
 serde = { version = "1", features = ["derive"] }
 sha2 = "0.8"
 sha3 = "0.8"
+strum = "0.18"
```

Comment:
Let's not add more dependencies, we are trying to keep `near-vm-logic` minimal because we have plans of compiling it to Wasm. Also `strum` is not a widely used library, let's implement enum iterator manually here.